src/gc/gcpriv.h

```diff
@@ -3002,6 +3002,17 @@ class gc_heap
     PER_HEAP_ISOLATED
     size_t min_segment_size;
 
+    // For SOH we always allocate segments of the same
+    // size unless no gc region requires larger ones.
```

Comment:
Yes. I knew what this comment was trying to say (we are, after all, in a PR about making changes in preparation for no-gc region), but my point is that it's ambiguous and needlessly hard to parse. Your suggestion about adding the underscores is fine.